app/main/views/suppliers.py

```diff
@@ -71,9 +63,9 @@ def suppliers_details(supplier_id):
     if any(supplier.get('service_counts', {}).get(framework_name, 0) > 0 for framework_name in live_framework_names):
         first_character_of_supplier_name = supplier["name"][:1]
         if is_alpha(first_character_of_supplier_name):
-            prefix = process_prefix(prefix=first_character_of_supplier_name, format='template')
+            prefix = first_character_of_supplier_name.upper()
         else:
-            prefix = u"other"
+            prefix = "other"
```

Comment:
Is this unnecessary use of formatter something a linter could detect?

Reply:
A linter could detect this (e.g. https://github.com/5j9/flake8-no-u-prefixed-strings). However, I think the risk of people introducing new instances of this is very low, so it might not be worth it.